src/service/ModifyMultisigService.ts

```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright 2020 NEM
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { prompt } from 'inquirer';
+import { Address, MultisigAccountModificationTransaction, NetworkType, Transaction, UnresolvedAddress } from 'symbol-sdk';
+import { LogType } from '../logger';
+import Logger from '../logger/Logger';
+import LoggerFactory from '../logger/LoggerFactory';
+import { Addresses, ConfigPreset } from '../model';
+import { AnnounceService, TransactionFactory, TransactionFactoryParams } from './AnnounceService';
+import { BootstrapUtils } from './BootstrapUtils';
+import { ConfigLoader } from './ConfigLoader';
+
+/**
+ * params necessary to announce multisig account modification transaction to network.
+ */
+export type ModifyMultisigParams = {
+    target: string;
+    password?: string;
+    url: string;
+    maxFee?: number | undefined;
+    useKnownRestGateways: boolean;
+    ready?: boolean;
+    customPreset?: string;
+    minRemovalDelta?: number;
+    minApprovalDelta?: number;
+    addressAdditions?: string;
+    addressDeletions?: string;
+    serviceProviderPublicKey?: string;
+};
+
+const logger: Logger = LoggerFactory.getLogger(LogType.System);
+
+export class ModifyMultisigService implements TransactionFactory {
+    public static readonly defaultParams: ModifyMultisigParams = {
+        target: BootstrapUtils.defaultTargetFolder,
+        useKnownRestGateways: false,
+        ready: false,
+        url: 'http://localhost:3000',
```

Comment:
default to hardcoded http?

Reply:
yes and default to localhost :)
Copied from: https://github.com/symbol/symbol-bootstrap/blob/56239efa3100116ea294c570ddab317160e0e00b/src/service/LinkService.ts#L77
I think it's fine for this PR, the node list loading logic will be changed after @fboucquez 's (soon to be raised) PR.

Reply:
Yes, it's fine, default url is when you are running the link/modify command on the same box that you have launched the node. Most people would either provide an `--url` or use `--useKnownRestGateways`